File: JS/D_Fixed_Password.js
const fs = require("fs");
const arr = fs.readFileSync(0, "utf8").trim().split(/\s+/).map(Number);

const pass = 1999;
for (let x of arr) {
    if (pass === x) {
        console.log("Correct");
        break;
    }
    else console.log("Wrong");
}

File: C/D_Fixed_Password.c
#include <stdio.h>

int main() 
{
    int password = 1999;
    while (1)
    {
        int num;
        scanf("%d", &num);
        if (num == password)
        {
            printf("Correct\n");
            break;
        }
        else printf("Wrong\n");
    }

    return 0;
}